Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -104,6 +104,10 @@ private enum PlatformToolset
 
         private readonly GUIContent uninstallLabel = new GUIContent("Uninstall First", "Uninstall application before installing");
 
+        private readonly GUIContent researchModeCapabilityLabel = new GUIContent("Enable Research Mode", "Enables research mode of HoloLens 1. This allows access to raw sensor data.");
+
+        private readonly GUIContent allowUnsafeCode = new GUIContent("Allow Unsafe Code", "Modify 'Assembly-CSharp.csproj' to allow use of unsafe code.");
```

Comment:
```suggestion
        private readonly GUIContent allowUnsafeCode = new GUIContent("Allow Unsafe Code", "Modify 'Assembly-CSharp.csproj' to allow use of unsafe code. Do not use this in production.");
```

Reply:
Hm...aren't there some cases were unsafe code is needed in production? So would it better to say "..unsafe code. Use this only in production if you know exactly what you are doing."  
And should i delete the 1 in line 107 from "...mode of HoloLens 1. This..."?

Reply:
Yeah your point is a good one - I think I just want to call out that people need to be careful with using this (i.e. Be careful using this in production).

Reply:
Went with your last suggestion and put "Be careful using this in production." at the end of the tooltip.